install/migrations/update_9.5.x_to_10.0.0/cache.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * ---------------------------------------------------------------------
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2015-2021 Teclib' and contributors.
+ *
+ * http://glpi-project.org
+ *
+ * based on GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * GLPI is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+/**
+ * @var DB $DB
+ * @var Migration $migration
+ */
+
+$had_custom_config = false;
+if (countElementsInTable('glpi_configs', ['name' => 'cache_db', 'context' => 'core'])) {
```

Comment:
What if somebody already has setted up a memcached/redis?

Reply:
I do not know if it is a good idea to try to migrate existing configurations, but it will require that we convert server/options from Laminas format to Symfony format. While this is not a big deal, it could be hard to test all cases and we may unintentionnally produce a wrong config.
Using filesystem cache by default for all has just a performance impact until cache configuration is put back in place. As current cache configuration has to be done directly in DB and is documented only on user documentation (not on install doc), I guess only a few people may have use it. So not migrating the configuration seems acceptable to me.

Reply:
I totally agree with you :)